sdk/go/pulumi-language-go/main.go

```diff
@@ -47,7 +47,7 @@ import (
 	pulumirpc "github.com/pulumi/pulumi/sdk/v3/proto/go"
 )
 
-func compileProgramCwd(buildID string) (string, error) {
+func compileProgramCwd(buildID string, outfile string) (string, error) {
```

Comment:
I should add a comment and repurpose the first string parameter as the outfile and not use it to salt the temp file that we delete anyway.